packages/ui-range-input/src/RangeInput/styles.js

```diff
@@ -0,0 +1,133 @@
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { DIRECTION } from '@instructure/ui-i18n'
+
+/**
+ * ---
+ * private: true
+ * ---
+ * Generates the style object from the theme and provided additional information
+ * @param  {Object} componentTheme The theme variable object.
+ * @param  {Object} props the props of the component, the style is applied to
+ * @param  {Object} state the state of the component, the style is applied to
+ * @return {Object} The final style object, which will be used in the component
+ */
+const generateStyle = (componentTheme, props, state) => {
+  const { size } = props
+  const { dir } = state
+  const rtlInputStyle = dir === DIRECTION.rtl ? { direction: 'rtl' } : {}
+  const valueSizeVariants = {
+    small: {
+      fontSize: componentTheme.valueSmallFontSize,
+      padding: componentTheme.valueSmallPadding,
+      lineHeight: componentTheme.valueSmallLineHeight
+    },
+    medium: {
+      fontSize: componentTheme.valueMediumFontSize,
+      padding: componentTheme.valueMediumPadding,
+      lineHeight: componentTheme.valueMediumLineHeight
+    },
+    large: {
+      fontSize: componentTheme.valueLargeFontSize,
+      padding: componentTheme.valueLargePadding,
+      lineHeight: componentTheme.valueLargeLineHeight
+    }
+  }
+  const trackStyle = {
+    borderRadius: '0.312em',
+    borderColor: 'transparent',
+    color: 'transparent',
+    cursor: 'pointer',
+    background: componentTheme.trackBackground,
+    height: `calc(${componentTheme.handleSize} / 2)`
+  }
+  const thumbStyle = {
+    appearance: 'none',
+    borderRadius: '50%',
+    cursor: 'pointer',
+    transition: 'all 0.15s ease-in-out',
+    width: componentTheme.handleSize,
+    height: componentTheme.handleSize,
+    background: componentTheme.handleBackground,
+    boxShadow: `0 0.0625rem 0 ${componentTheme.handleShadowColor}`,
+    '&:hover': {
+      background: componentTheme.handleHoverBackground
+    }
+  }
+  const thumbFocusActiveStyle = {
+    boxShadow: `0 0.0625rem 0 ${componentTheme.handleShadowColor}, 0 0 0 ${componentTheme.handleFocusOutlineWidth} ${componentTheme.handleFocusOutlineColor}`
+  }
+
+  return {
+    rangeInput: {
+      label: 'rangeInput',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      minWidth: componentTheme.minWidth
+    },
+    rangeInputInput: {
+      label: 'rangeInput__input',
+      all: 'initial',
+      flex: 'auto',
+      boxSizing: 'border-box',
+      appearance: 'none',
+      width: '100%', // for Firefox
+      outline: 'none',
+      margin: 0,
+      '&::-webkit-slider-thumb': {
+        ...thumbStyle,
+        marginTop: `calc(-1 * ${componentTheme.handleSize} / 4)`
+      },
+      '&::-moz-range-thumb': thumbStyle,
+      '&:focus, &:active': {
+        outline: 'none',
+        '&::-webkit-slider-thumb': thumbFocusActiveStyle,
+        '&::-moz-range-thumb': thumbFocusActiveStyle
```

Comment:
for some reason this was not working when I defined in one style ( `'&::-webkit-slider-thumb &::-moz-range-thumb': ...`

Reply:
figured out: its because the old code used the `postcss-input-range` plugin